lib/lita/handlers/puppet.rb

```diff
@@ -35,6 +35,15 @@ class Puppet < Handler
       )
 
       route(
+        /(puppet|pp)\s+(class)\s+(nodes)\s+(\S+)/i,
+        :class_nodes,
```

Comment:
This is the name of the method _in this class_ that the route points to. So you should change this to `:nodes_with_class`.